src/assets/AssetsContractController.ts

```diff
@@ -78,11 +102,21 @@ export class AssetsContractController extends BaseController<
     this.defaultConfig = {
       provider: undefined,
       ipfsGateway: IPFS_DEFAULT_GATEWAY_URL,
+      chainId: SupportedTokenDetectionNetworks.mainnet,
     };
     this.initialize();
+
     onPreferencesStateChange(({ ipfsGateway }) => {
       this.configure({ ipfsGateway });
     });
+
+    onNetworkStateChange((networkState) => {
```

Comment:
Added test that calls `getBalancesInSingleCall` after switching network. Under the hood, it will call this function to update `chainId` on `AssetsContractController`

Reply:
Cool. I think ideally we should have a unit test just for this that doesn't involve another method, so it is a little more obvious when going through the tests, but I don't think that is a blocker for this. Thanks!